controllers/controller.js

```diff
@@ -22,3 +22,9 @@ exports.patchReviewById = (req, res, next) => {
     })
     .catch(next)
 }
+
+exports.getUsers = (req,res) => {
+    selectUsers().then((user) => {
+        res.status(200).send({ user })
+    })
```

Comment:
You should put a `.catch` block on all of your controllers to stop any unhandled errors crashing your server. You're not expecting any but it still be protected against